src/app.py

```diff
@@ -1,40 +1,64 @@
-import logging
+import logging, coloredlogs, yaml, os, sys, urllib3, requests
```

Comment:
non-functional minor things/points:
Although Python aloughs comma separated imports, they're not often used in the wild - 
Python's own coding standard (that a lot of folks just adhere to) recommends one import statement per line.

Its also generally good practice/standard Python coding convention (might even be in PEP8) to organise imports like:
- builtin modules
<newline separator>
- third-party modules
<newline separator>
- local project modules

Makes it easy to at a glance where the module is from without searching.